go.sum

```diff
@@ -5,12 +5,14 @@ github.com/census-instrumentation/opencensus-proto v0.2.1/go.mod h1:f6KPmirojxKA
 github.com/client9/misspell v0.3.4/go.mod h1:qj6jICC3Q7zFZvVWo7KLAzC3yx5G7kyvSDkc90ppPyw=
 github.com/davecgh/go-spew v1.1.0/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
 github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b/go.mod h1:SBH7ygxi8pfUlaOkMMuAQtPIUF8ecWP5IEl/CR7VP2Q=
+github.com/golang/groupcache v0.0.0-20190702054246-869f871628b6 h1:ZgQEtGgCBiWRM39fZuwSd1LwSqqSW0hOdXCYYDX0R3I=
```

Comment:
Where did the dependency on groupcache come from?

Reply:
`groupcache` is a dependency for `opencensus` library. The entry for the module was already present however the checksum entry was not. When I added the `go-cmp` module, the missing checksum entry for `groupcache` was added

https://github.com/orijtech/ocagentconv/blob/01290221e769ffb61bacf789bd9ea078320e596c/go.sum#L8

Reply:
Gotcha, thanks!